src/components/Event/styles.js

```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  flex-wrap: wrap;
+  display: flex;
+  justify-content: space-between;
+  width: 300px;
+  overflow: hidden;
+  box-shadow: 0px 0px 15px -5px;
+  transition: 0.5s;
+  animation: ease-in;
+  margin: 15px;
+  border-radius: 15px;
+
+  &:hover {
+    transform: scale(1.1);
+    box-shadow: 0px 0px 15px 0px;
+  }
+`;
+
+export const Content = styled.div`
+  margin: 1rem;
+  margin-top: 0.5rem;
+`;
+
+export const ImageContainer = styled.div`
+  overflow: hidden;
+  display: flex;
+  height: 220px;
+  width: 300px;
+
+  & > img {
+    flex: 1;
+    width: 300px;
+    height: 220px;
+  }
+`;
+
+export const CardTitle = styled.div`
+  color: ${(props) => props.theme.hue.blue};
+  text-transform: capitalize;
+  margin-top: -10px;
+  margin-bottom: -10px;
+`;
+
+export const CardDate = styled.a`
```

Comment:
Don't believe that this should be an address. Maybe change this to p or div so it matches with what you have inside index.js